mlir/test/e2e/PaddedGemmConfig.toml

```diff
@@ -0,0 +1,48 @@
+directory = "PaddedGemm"
```

Comment:
Do you mind to get rid of any redundant configs that were sitting under auto-e2e directory? I think you have provided a fairly complete set of test in here.

Reply:
Done.